docs/features/sessions/pause.md

```diff
@@ -41,3 +41,8 @@ This is so that you don't lose your work!
 When you have added credits, just resume the session and carry on with your work. The left navigation bar always shows credits remaining.
 
 ![](/images/sessions/autopausing.gif)
+
+## Pause a Session Using a Github Action
+
+When you are not using your session, You can pause it using the Github Action.
```

Comment:
This language could be better. Approved none the less.

For example this reads better:

"We provide an example GitHub Action [template](https://github.com/sunitaprakash/grid-session-env) for pausing Sessions"

Reply:
I will be making the language change then merging.